netstandard/ref/System.cs

```diff
@@ -262,7 +261,7 @@ public sealed partial class ApplicationId
         public override int GetHashCode() { throw null; }
         public override string ToString() { throw null; }
     }
-    public partial class ArgumentException : System.SystemException, System.Runtime.Serialization.ISerializable
```

Comment:
Why are we removing serialization here? Mostly surprised it was there to be removed.

Reply:
Because `Exception` now implements `ISerializable`